frontend/src/features/properties/hooks/useParcelLayerData.tsx

```diff
@@ -11,12 +11,38 @@ import { RootState } from 'reducers/rootReducer';
 import useCodeLookups from 'hooks/useLookupCodes';
 import { toast } from 'react-toastify';
 import useDeepCompareEffect from 'hooks/useDeepCompareEffect';
+import { getInitialValues } from 'features/mapSideBar/SidebarContents/LandForm';
 
 interface IUseParcelLayerDataProps {
   formikRef: React.MutableRefObject<FormikValues | undefined>;
   parcelId?: number | '';
+  nameSpace?: string;
 }
 
+/**
+ * Only autofill data if:
+ * 1) layerdata and formikref are bot non-null/undefined
+ * 2) either the layer pid or pin has a non-zero value
+ * 3) the pid or pin are not equal to the initial values (indicating that the form has already been pre-populated)
+ * @param layerData
+ * @param formikRef
+ * @param nameSpace
+ */
+const isFormInStateToSetLayerData = (
```

Comment:
moved this if statement out due to its complexity. This prevents duplicate requests when the pid input is set to the most recent results of a pid search.